### ISSUE TITLE
add portable exe artifact

### ISSUE THREAD

**Issue by art0007i** [Gawdl3y/Resolute#58]
I don't need an installer for a single self-contained exe file

**Comment by art0007i** [comment]
wait does the one I installed with the setup.exe self update? does that just mean if I download the setup exe of a new version it will auto install in the correct location or will it completely autonomously just update itself?

**Comment by Gawdl3y** [comment]
Yes, the installed versions are self-updating. At launch, it checks for an update and prompts you for whether you'd like to install it if there is one available and shows you the release notes.
